<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recuperar_Senha</class>
 <widget class="QWidget" name="Recuperar_Senha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Recuperar_Senha{
	background-color: rgb(0, 170, 255);
}

QFrame#frame{
	background-color: white;
	border-radius: 10px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QFrame" name="frame">
       <property name="maximumSize">
        <size>
         <width>450</width>
         <height>250</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="5" column="0">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QPushButton" name="button_voltar">
              <property name="minimumSize">
               <size>
                <width>135</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>135</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color:rgb(24, 136, 255);
	color: black;
	font-weight: 700;
	border-radius: 6px;
}
QPushButton:hover{
	background-color: rgb(98, 145, 255);
}</string>
              </property>
              <property name="text">
               <string>Voltar</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
             <widget class="QLabel" name="label_erro">
              <property name="styleSheet">
               <string notr="true">font-weight: bold;
color: red;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">font-weight: bold;
font-size: 12px;</string>
              </property>
              <property name="text">
               <string>E-mail:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="line_email">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>350</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:rgb(211, 225, 255);
border-radius: 3px;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignTop">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font-weight: bold;
font-size: 17px;</string>
            </property>
            <property name="text">
             <string>Recuperar Senha</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QPushButton" name="button_recuperar_senha">
              <property name="minimumSize">
               <size>
                <width>135</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>135</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color:rgb(24, 136, 255);
	color: black;
	font-weight: 700;
	border-radius: 6px;
}
QPushButton:hover{
	background-color: rgb(98, 145, 255);
}</string>
              </property>
              <property name="text">
               <string>Recuperar Senha</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">font-weight: bold;
font-size: 12px;</string>
              </property>
              <property name="text">
               <string>Username:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="line_user">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>350</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:rgb(211, 225, 255);
border-radius: 3px;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>line_user</tabstop>
  <tabstop>line_email</tabstop>
  <tabstop>button_recuperar_senha</tabstop>
  <tabstop>button_voltar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
